README.md

```diff
@@ -220,6 +220,39 @@ physeq
 ## phy_tree()    Phylogenetic Tree: [ 759 tips and 757 internal nodes ]
 ```
 
+## Creating a TreeSummarizedExperiment (TSE) Object
+
+A wrapper function called `qza_to_tse()` is provided which links multiple `read_qza()` calls together to create a TSE object for subsequent analysis. You may find some useful tutorials on :
+[OMA tutorials](https://jserizay.com/OMA/docs/devel/). An example usage is shown below:
+
+```
+tse<-qza_to_tse(
+    features="inst/artifacts/2020.2_moving-pictures/table.qza",
+    tree="inst/artifacts/2020.2_moving-pictures/rooted-tree.qza",
+    taxonomy="inst/artifacts/2020.2_moving-pictures/taxonomy.qza",
+    metadata = "inst/artifacts/2020.2_moving-pictures/sample-metadata.tsv"
+    )
```

Comment:
I am not sure if this works when you *install* the qiime2R package; you can try in a fresh session to do devtools::load_all("/my/path/qiime2R") and then try to run this example. Doesn't work?

You should probably refer to something like
 file_path <- system.file("inst/artifacts", "mydatafilename.txt", package = "qiime2R")